test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +57,12 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertNoNull(Object... objects) {
```

Comment:
You can consider just using assertFalse.(Utils.isAnyNull(...)) in the method isAnyNull on Line 14 instead of abstracting this out as a method. (... contains what you want to test.)

The reason why existing tests do that (assertAreUnique and assertNotUnique) is to avoid repeating usage of Arrays.asList.

Same comment for the method `assertHasNull`.